src/charm.py

```diff
@@ -28,6 +28,8 @@
 class SpringBootCharm(CharmBase):
     """Spring Boot Charm service."""
 
+    on = CharmEvents()
```

Comment:
I think `DCO060` was added to the ignore list because it was a bit over zelous. It has since been pared back, would it be worth it to re-enable the rule and document class attributes in the docstring?

Reply:
Fixed